src/plugins/mer/meremulatormodedetailswidget.ui

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Mer::Internal::MerEmulatorModeDetailsWidget</class>
+ <widget class="QWidget" name="Mer::Internal::MerEmulatorModeDetailsWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>824</width>
+    <height>463</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout_2">
```

Comment:
The top level layout needs all margins set to zero.

Reply:
Fixed.